RequestKit.xcodeproj/project.pbxproj

```diff
@@ -867,6 +868,7 @@
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				SKIP_INSTALL = YES;
 				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
+				SWIFT_VERSION = 3.0;
```

Comment:
ah I see, thanks so much for the instructions! I hit backspace and the whole `SWIFT_VERSION` entry in the project build settings was removed, I guessthat was the desired outcome?!